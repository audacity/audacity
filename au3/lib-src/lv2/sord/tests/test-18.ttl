@prefix : <http://example.org/foo#> .

:a :b """\nthis \ris a \U00015678long\t
literal\uABCD
""" .

:d :e """\tThis \uABCDis\r \U00015678another\n
one
""" .
