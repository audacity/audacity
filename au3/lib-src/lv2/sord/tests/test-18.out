<http://example.org/foo#a> <http://example.org/foo#b> "\nthis \ris a \U00015678long\t\nliteral\uABCD\n" .
<http://example.org/foo#d> <http://example.org/foo#e> "\tThis \uABCDis\r \U00015678another\n\none\n" .
